Assets/MixedRealityToolkit/Services/MixedRealityToolkit.cs

```diff
@@ -648,7 +648,6 @@ private void InitializeInstance()
                     if (instance != null)
                     {
                         Debug.Assert(instance.transform.parent == null, "The MixedRealityToolkit should not be parented under any other GameObject!");
-                        Debug.Assert(instance.transform.childCount == 0, "The MixedRealityToolkit should not have GameObject children!");
```

Comment:
> Debug.Assert(instance.transform.childCount == 0, "The MixedRealityToolkit should not have GameObject children!"); [](start = 23, length = 114)

Is it possible to assert that there isn't any non-facade child? I guess this is pretty minor because the user will see it get nuked anyway right.

Reply:
Yeah it should be pretty self-evident what's going on. I'll be eager to hear customer feedback on this one, though.